README.md

```diff
@@ -75,7 +75,25 @@ If you want to use the `remote` mode, you have to specify all authorization keys
             tenantId: "your-tenant-id"
     }
 }
-````
+```
+
+You can also include remote projects if you want to use remote media.
+
+For this you can add another parameter called `remotes` to the config. There it expects an object, which requires a unique name as key and the corresponding project id as value. For example:
```

Comment:
... The request expects .... project-id ...

Reply:
Actually the request does not expect an object, but the parameter does. I changed it anyway to be more clear about this.